boards/jiminy-mega256rfr2/include/periph_conf.h

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (C) 2016 RWTH Aachen, Josua Arndt
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_jiminy-mega256rfr2
+ * @{
+ *
+ * @file
+ * @brief       Peripheral MCU configuration for the Jiminy Mega 256rfr2 board
+ *
+ * @author      Josua Arndt <jarndt@ias.rwth-aachen.de>
+ * @author	Steffen Robertz <steffen.robertz@rwth-aachen.de>
```

Comment:
author name misaligned.
I guess issues found in previous comments from `board.h` needs to be fixed as well in this file